atom/browser/api/atom_api_web_contents.cc

```diff
@@ -309,13 +311,23 @@ void WebContents::DidStopLoading(content::RenderViewHost* render_view_host) {
 
 void WebContents::DidGetResourceResponseStart(
     const content::ResourceRequestDetails& details) {
+  net::HttpResponseHeaders* headers = details.headers.get();
+  base::DictionaryValue response_headers;
```

Comment:
:+1:  these callbacks are threadsafe right, I dont have to lock `v8::Isollate` before the dictionary creation ?